os-3.10/scripts/provision.sh

```diff
@@ -109,3 +109,18 @@ ansible-playbook -i $inventory_file $openshift_ansible/playbooks/deploy_cluster.
 /usr/bin/oc create identity allow_all_auth:admin
 /usr/bin/oc create useridentitymapping allow_all_auth:admin admin
 /usr/bin/oc adm policy add-cluster-role-to-user cluster-admin admin
+
+# Create local-volume directories
+for i in {1..10}
```

Comment:
Does this mean that I can create 10 `PVC`s with the storage class `local` per node?

Reply:
Correct, so in a 3 node cluster it will create 30 PVs, and when you request a PVC, it will bind one of the available ones, and the pod claiming it will be scheduled on the node the PV that was claimed is defined on. Also note I marked the storage class as default, so if you don't specify a storage class it will use the local volume one.

Reply:
Actually you can add more PVs if you want, by simply mount binding more directories after the fact (like I described in the original documentation PR that spawned this one). I figured 10 PVs per node would a decent default, but you have to ability to expand if needed.

Reply:
Great. Thx for the explanation.